pipestat/backends/db_backend/dbbackend.py

```diff
@@ -467,6 +502,105 @@ def select_records(
 
         return records_dict
 
+    def retrieve_history_db(self, record_identifier, result_identifier=None):
+        """
+
+        :param record_identifier: single record_identifier
+        :param result_identifier: single or list of result identifiers
+        :return: dict records_dict = {
+            "history": List[Dict[{key, Any}]],
+        }
+        """
+
+        record_identifier = record_identifier or self.record_identifier
+
+        ORMClass = self.get_model(table_name=self.table_name)
+        ORMClass_History = self.history_table[list(self.history_table.keys())[0]]
+
+        if not result_identifier:
+            columns = None
+        else:
+            if isinstance(result_identifier, str):
+                columns = [result_identifier]
+            elif isinstance(result_identifier, list):
+                columns = result_identifier
+            else:
+                raise ValueError("Result identifier must be a str or list[str]")
+            for i in ["id", MODIFIED_TIME]:
+                if i not in columns:
+                    columns = [i] + columns
```

Comment:
please use append if possible

Reply:
I changed this to `columns.insert(0, i)`. However, I had to create a copy of the list initially, anyway, otherwise, the original list becomes modified and it causes problems when the results are returned.